### DISCUSSION TITLE
2023.06.29 - Community Meeting

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#8316] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to held Bimonthly Community Meeting where we can share main updates and also have an open discussion with you.

---

**💥Eleventh Community Meeting will take place on Thursday 29/06/2023 at 4 pm GMT+1 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you already participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you join us again or not.

---

### Agenda will be specified below in the comments:  (feel free to add any topic or idea)

1) Development updates & plans
- 2.0.10 release
- 3.0.0 release update
- 3.0.0 release plans

2) Community Topics
- Issues - Diagnostic report​ (developed by @me-no-dev​)
- [Draft: secrets #8310](https://github.com/espressif/arduino-esp32/pull/8310)

3) Open Discussion

---

See you on the Meeting!

**Comment by brentru** [comment]
For Community Topics:
Secrets draft: https://github.com/espressif/arduino-esp32/pull/8310

**Reply by VojtechBartoska** [reply]
Thanks for the review of that particular PR, adding this to the agenda.

**Comment by HeFeng1947** [comment]
For Community Topics:
ESP will soon be officially mass producing the ESP32-H2. [Adafruit has already released a development board for the H2](https://www.adafruit.com/product/5715). How is support for the esp-h2 and thread progressing?

**Reply by VojtechBartoska** [reply]
Hi, adding this to the Agenda, we will cover new SoCs support status.

**Comment by VojtechBartoska** [comment]
Slides are available here:
[11_Eleventh_Community_meeting-Arduino-June_20231.pdf](https://github.com/espressif/arduino-esp32/files/11906967/11_Eleventh_Community_meeting-Arduino-June_20231.pdf)
